src/test/java/pl/techlab24/OSKManager/model/validation/AlertValidatorTest.java

```diff
@@ -0,0 +1,122 @@
+package pl.techlab24.OSKManager.model.validation;
```

Comment:
General comment to all tests:
add special test's comments:
// given
// when
// then
to all parameterized test methods